### ISSUE TITLE
Generating modules

### ISSUE THREAD

**Issue by mmenavas** [hechoendrupal/DrupalAppConsole#286]
I was getting an error message because I didn't realize that to generate a module I had to run the console commands from a drupal installation.

![standalone_console](https://cloud.githubusercontent.com/assets/770699/5668842/ae7adc78-9740-11e4-9e1b-d2766a0b19e8.png)

I think this exception should be caught and replaced with a more helpful message.

**Comment by dmouse** [comment]
You are into the drupal 8 folder?

**Comment by mmenavas** [comment]
No, I wasn't. That information was missing on http://drupalconsole.com/.

**Comment by mmenavas** [comment]
Maybe it's obvious for a lot of people, but a message informing that I need to be in a drupal directory is more helpful than throwing an exception.

**Comment by vacho** [comment]
indeed matter if you dare to run 
$ drupal generate:module 
anywhere else other than the drupal root also get an error that does little to inexperienced.

**Comment by jmolivas** [comment]
I am planning to fix both as:

* Without calling a command
```
$ drupal
```
* Calling a command
```
$ drupal generate:module 
```
![drupal-outside-drupal-root](https://cloud.githubusercontent.com/assets/366275/5684949/a1a5f866-97ea-11e4-8fc6-e251cd443a32.png)

**Comment by jmolivas** [comment]
Update console to show commands before the error message 

![drupal-outsideroot-final](https://cloud.githubusercontent.com/assets/366275/5693018/568e9b4a-98c0-11e4-9ead-11207f1c3a50.png)

**Comment by jmolivas** [comment]
Fixed on #287
Message
 ```
[Exception]
  In order to list all of the available commands, you should run this inside
  a drupal root directory.
```
![drupal-outsideroot-final](https://cloud.githubusercontent.com/assets/366275/5729845/73a980e8-9b2a-11e4-8f23-f52c883d07c3.png)

The message may be changes form `Exception` to `Notice` or something more user friendly later
